README.md

```diff
@@ -92,6 +92,7 @@ The following options are available:
       "faiUrl": "https://aveit.s3.amazonaws.com/higlass/data/sequence/hg38.fa.fai",
       "chromSizesUrl": "https://aveit.s3.amazonaws.com/higlass/data/sequence/hg38.mod.chrom.sizes"
     },
+    "showTooltip": false, // If true, mouseover events over transcripts show an information tooltip
```

Comment:
In the [higlass-multivec](https://github.com/higlass/higlass-multivec/pull/82) PR, this defaults to true. Should it be consistent across the two?

Reply:
To be consistent, I will set it `true` here.